feat: :sparkles: Criando rota de exclusão

Layravbf opened the pull request:
Criando rota de exclusão de usuários, adopsteam e campanhas

**What issue does this pull request resolve?**

**What changes did you make?**

**Is there anything that requires more attention while reviewing?**

Layravbf committed:
feat: :sparkles: Criando rota de exclusão

Criando rota de exclusão de usuários, adopsteam e campanhas

LucasTonetto commented on src/ts/models/DAO/AdOpsTeamDAO.ts:
```diff
@@ -92,6 +92,22 @@ export class AdOpsTeamDAO {
+	 * @param adOpsTeamId AdOpsTeam que será deletado
+	 * @returns Retorna True caso o adOpsTeam tenha sido deletado com sucesso
+	 */
+	public deleteAdopsteam(adOpsTeamId: string): Promise<boolean | void> {
```
SEmpre vai retornar um boolean, deixar o retorno como Promise<boolean>

LucasTonetto commented on src/ts/models/cloud/FirestoreConnectionSingleton.ts:
```diff
@@ -140,6 +140,16 @@ export class FirestoreConnectionSingleton extends ObjectStore {
+	 * @param id ID do documento
+	 * @returns Documento deletado
+	 */
+	public deleteDocumentById(collection: CollectionReference, id: string): Promise<WriteResult> {
```
Faltou criar a rota para exclusão de Adopsteam, campanha e usuario

Layravbf committed:
feat: :sparkles: Correção do retorno da promise e criação das rotas

retirando void do retorno da promise e criando as rotas de delete para usuario, campanhas e adopsteam

Layravbf committed:
feat: :sparkles: Incluindo a verificação de adOpsManager deletar somente usuários do próprio time

Layravbf committed:
style: :lipstick: Apply formatting changes

Layravbf committed:
fix: :bug: Npm run build para gerar o arquivo .js alterado

Npm run build para gerar o arquivo .js alterado
